.travis.yml

```diff
@@ -14,7 +14,7 @@ install:
   - conda config --set always_yes yes --set changeps1 no
   - conda update -q conda
   - conda info -a
-  - conda create -q -n test-environment -c conda-forge python=$TRAVIS_PYTHON_VERSION pandas dask numba numpy "pyarrow>=0.15" pytest hypothesis scipy shapely geopandas
+  - conda create -q -n test-environment -c conda-forge python=$TRAVIS_PYTHON_VERSION pandas dask numba numpy "pyarrow>=0.15" pytest hypothesis scipy shapely geopandas param
   - conda activate test-environment
   # hilbertcurve not available with conda, install with pip
```

Comment:
I contributed hilbertcurve yesterday, you can now also conda install it.

Reply:
Oh awesome. Thanks!